migration/migrations/001_initial.py

```diff
@@ -0,0 +1 @@
+print("Initial Migration")
```

Comment:
please nest all the migration scripts in another directory, i.e. `migration/versions/*.py`.  this will make it easier on the Migration class to find and sort them.